pip/worker/pypi.go

```diff
@@ -105,7 +105,7 @@ func NewPypiPackageDataFactory(client *helper.Client) PypiPackageDataFactory {
 func (pf *pypiPackageDataFactory) GetPackageData(packageJSONURL string) (PypiPackageData, error) {
 	packageInfo := PypiPackageData{}
 
-	packageJSONURL = strings.Replace(packageJSONURL, "pypi.org", "", 1)
+	packageJSONURL = strings.ReplaceAll(packageJSONURL, "pypi.org", "")
```

Comment:
Could you edit your commit to explain why this change is needed?

Reply:
Since the http client was changed to accept a baseURL, so that external http requests can be mocked, I had to strip off `pypi.org` from the base url and hence in the first commit, I had introduced
 ```
packageJSONURL = strings.Replace(packageJSONURL, "pypi.org", "", 1)
 ```
which was later replaced by just to abide by the code style of the codebase.

```
packageJSONURL = strings.ReplaceAll(packageJSONURL, "pypi.org", "")
```

I thought the `chore` prefix to the commit made sense, but I'm open to adding a comment or changing the commit message for more verbosity.